src/popup/router/components/dashboard/Card.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div
+    class="card"
+    :class="{ 'is-big-card': isBigCard, hover }"
+    :style="styleComponent"
+    @click="(e) => $emit('click', e)"
+  >
+    <div class="card-icon">
+      <slot
+        name="icon"
+        class="icon-svg"
+      />
+    </div>
+    <div>
+      <div class="title">
+        {{ title }}
+      </div>
+      <div class="description">
+        {{ description }}
+      </div>
+      <slot />
+    </div>
+  </div>
+</template>
+
+<script>
+
+export default {
+  name: 'Card',
+  props: {
+    title: {
+      type: String,
+      default: '',
+      required: true,
+    },
+    description: {
+      type: String,
+      default: '',
+      required: true,
+    },
+    background: {
+      type: String,
+      default: null,
+    },
+    isBigCard: Boolean,
+    hover: Boolean,
```

Comment:
I think the name of the prop is misleading. Can be anything. I suggest to rename it to `hoverable`, `has-hover`, `has-hover-effet`, `is-interactive`, `clickable`... Choose any :)